tests/scripts/run-throughput-perf.py

```diff
@@ -32,31 +32,20 @@
 # Globals
 ##########################################################################
 
-# List of dlls we want to crossgen
-dll_list = {
+# List of dlls we want to exclude
+dll_exclude_list = {
     'Windows_NT': [
-        "System.Private.CoreLib",
-        "System.Reflection.Metadata",
-        "System.Linq.Expressions",
-        "Microsoft.CSharp",
-        "System",
-        "Microsoft.CodeAnalysis.VisualBasic",
-        "System.Private.DataContractSerialization",
-        "System.Core",
-        "System.Xml",
-        "Microsoft.CodeAnalysis.CSharp",
-        "Microsoft.CodeAnalysis",
-        "System.Linq.Parallel",
-        "System.Private.Xml"
+        # Require Newtonsoft.Json
```

Comment:
The comment is why it's being excluded. Everything below the comment, and before the next comment is excluded because they require Newtonsoft.Json to crossgen, which is not found in the packaged up assemblies we are using. All items on the exclude list require additional assemblies not found in the core_root dir that I used when I created the throughput nuget package.